src/Highlighter.php

```diff
@@ -52,6 +52,11 @@ public function __construct(
         }
     }
 
+    public function isSupportedLanguage(string $language): bool
```

Comment:
Given the naming of the other methods, I wonder if it would be more intuitive to call this `hasLanguage`?

Reply:
ok. done.